index.html

```diff
@@ -7176,6 +7414,334 @@ <h2 id="MediaStreamAudioSourceNode">
         </dl>
       </section>
     </section>
+    <section>
+      <h2>
+        Processing model
+      </h2>
+      <section class="informative">
+        <h3>
+          Background
+        </h3>
+        <p>
+          Real-time audio systems that require low latency are, often time,
+          implemented using <em>callback functions</em>, where the operating
+          system calls the program back when more audio has to be computed in
+          order for the playback to stay uninterrupted. Such callback is called
+          on a high priority thread (often the highest priority on the system).
+          This means that a program that deals with audio only executes code
+          from this callback, as any buffering between a rendering thread and
+          the callback would naturally add latency or make the system less
+          resilient to glitches.
+        </p>
+        <p>
+          For this reason, the traditional way of executing asynchronous
+          operations on the Web Platform, the event loop, does not work here,
+          as the thread is no <em>continuously executing</em>. Additionally, a
+          lot of unnecessary and potentially blocking operations are available
+          from traditional execution contexts (Windows and Workers), which is
+          not something that is desirable to reach an acceptable level of
+          performance.
+        </p>
+        <p>
+          Additionally, the Worker model makes creating a dedicated thread
```

Comment:
Yes, Workers are on their own thread, separate from the control and rendering threads.